.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "printWidth":  120
+}
```

Comment:
Please, if you don't mind, add a new empty line in order to satisfy Github requirements :smile: